recipe/build.sh

```diff
@@ -141,6 +141,7 @@ fi
         --disable-static \
         --enable-version3 \
         --enable-zlib \
+        --enable-libopus \
```

Comment:
Worth noting (for the context of reviewing, because I had to check myself) that with `bld.bat` wrapping `build.sh`, this is effective on windows as well.

Reply:
i had forgotten about this too!